docs/source/backends/raspbian.rst

```diff
@@ -0,0 +1,683 @@
+                    "ssid": "freifunk",
+                    "mode": "adhoc",
+                    "bssid": "02:b8:c0:00:00:00"
+                }
+            }
+        ]
+    }
+
+Will result in::
+
+    config: /etc/network/interfaces
+    auto wireless
+    iface wireless inet static
+        address 172.128.1.1
+        netmask 255.255.255.0
+        wireless-channel 1
+        wireless-essid freifunk
+        wireless-mode ad-hoc
+
+
+WPA2 Personal (Pre-Shared Key)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows a typical wireless access
+point using *WPA2 Personal (Pre-Shared Key)* encryption:
+
+.. code-block:: python
+
+    {
+        "radios": [
+            {
+                "name": "radio0",
+                "phy": "phy0",
+                "driver": "mac80211",
+                "protocol": "802.11n",
+                "channel": 3,
+                "channel_width": 20,
+                "tx_power": 3
+            }
+        ],
+        "interfaces": [
+            {
+                "name": "wlan0",
+                "type": "wireless",
+                "wireless": {
+                    "radio": "radio0",
+                    "mode": "access_point",
+                    "ssid": "wpa2-personal",
+                    "encryption": {
+                        "protocol": "wpa2_personal",
+                        "cipher": "tkip+ccmp",
+                        "key": "passphrase012345"
+                    }
+                }
+            }
+        ]
+    }
+
+Will be rendered as follows::
+
+    config: /etc/hostapd/hostapd.conf
+    interface=wlan0
+    driver=nl80211
+    hw_mode=g
+    channel=3
+    ieee80211n=1
+    ssid=wpa2-personal
+    auth_algs=1
+    wpa=2
+    wpa_key_mgmt=WPA-PSK
+    wpa_passphrase=passphrase012345
+    wpa_pairwise=TKIP CCMP
+
+NTP settings
+------------
+
+The Network Time Protocol settings reside in the ``ntp`` key of the
+*configuration dictionary*, which is a custom NetJSON extension not present in
+the original NetJSON RFC.
+
+The ``ntp`` key must contain a dictionary, the allowed options are:
+
++-------------------+---------+---------------------+
+| key name          | type    | function            |
++===================+=========+=====================+
+| ``enabled``       | boolean | ntp client enabled  |
++-------------------+---------+---------------------+
+| ``enable_server`` | boolean | ntp server enabled  |
++-------------------+---------+---------------------+
+| ``server``        | list    | list of ntp servers |
++-------------------+---------+---------------------+
+
+NTP settings example
+~~~~~~~~~~~~~~~~~~~~
+
+The following *configuration dictionary* :
+
+.. code-block:: python
+
+    {
+        "ntp": {
+        "enabled": True,
+        "enable_server": False,
+        "server": [
+            "0.openwrt.pool.ntp.org",
+            "1.openwrt.pool.ntp.org",
+            "2.openwrt.pool.ntp.org",
+            "3.openwrt.pool.ntp.org"
```

Comment:
use a different pool of servers otherwise it looks badly copied from the openwrt backend page :-P